examples/networking/xdp/xdp_drop_count.py

```diff
@@ -0,0 +1,123 @@
+from bcc import BPF
+import time
+import sys
+
+
+
+
+
+if (len(sys.argv) != 2):
+    print("Usage: {0} <ifdev>\n\ne.g.: {0} eth0".format(sys.argv[0]))
+    exit(1)
+
+
+device = sys.argv[1]
+
+
+# load BPF program
+b = BPF(text = """
+    #include <uapi/linux/bpf.h>
+    #include <linux/in.h>
+    #include <linux/if_ether.h>
+    #include <linux/if_packet.h>
+    #include <linux/if_vlan.h>
+    #include <linux/ip.h>
+    #include <linux/ipv6.h>
+
+
+    BPF_TABLE("percpu_array", uint32_t, long, dropcnt, 256);
+
+
+    static inline int parse_ipv4(void *data, u64 nh_off, void *data_end)
+    {
+        struct iphdr *iph = data + nh_off;
+
+        if ((void*)&iph[1] > data_end)
+            return 0;
+        return iph->protocol;
+    }
+
+    static inline int parse_ipv6(void *data, u64 nh_off, void *data_end)
+    {
+        struct ipv6hdr *ip6h = data + nh_off;
+
+        if ((void*)&ip6h[1] > data_end)
+            return 0;
+        return ip6h->nexthdr;
+    }
+
+
+
+    int xdp_prog1(struct xdp_md *ctx) {
```

Comment:
Opening braces here are inconsistent. In bcc, we're mostly using same-line opening braces, meaning this line is correct and the `parse_ipv[46]` functions should be compacted by 1 line.